src/Keystore/EncryptionParameters.cpp

```diff
@@ -28,6 +28,8 @@ static Data computeMAC(Iter begin, Iter end, const Data& key) {
     return Hash::keccak256(data);
 }
 
+#pragma GCC diagnostic ignored "-Wunused-variable"
```

Comment:
I think we should throw an error instead of assert, if encryption failed, better to stop immediately

Reply:
This is in constructor, no return value, only way is to throw exception.  And a negative test case also would be needed for it, I'll check if there is a feasible way to make encryption fail.

Reply:
The only way aes_encrypt_key() can fail is with wrong key size, but we use constant 16.  I think asser() only is enough.